mdata/ifaces.go

```diff
@@ -12,6 +13,12 @@ type Metrics interface {
 
 type Metric interface {
 	Add(ts uint32, val float64)
-	Get(from, to uint32) (uint32, []chunk.Iter)
-	GetAggregated(consolidator consolidation.Consolidator, aggSpan, from, to uint32) (uint32, []chunk.Iter)
+	Get(from, to uint32) MetricResult
+	GetAggregated(consolidator consolidation.Consolidator, aggSpan, from, to uint32) MetricResult
+}
+
+type MetricResult struct {
```

Comment:
It's named "Raw" because there are raw points, while the "Iters" contains compressed points. Naming it "Points" might be confusing because both of them contain points, the difference is just that one of them has them raw and the other has them compressed. But yeah, then it should be called "Raw" and "Compressed":) I can also name it points and iters, that's fine anyway